dev/dram/device.c

```diff
@@ -7,23 +7,45 @@
 #include <compiler.h>
 #include <dram/device.h>
 
-#define CA_LAST         (3)
+#define CS_LOW                  (0)
+#define CS_HIGH                 (1)
+#define CA_CLK_LEN              (3)
+#define WRITE_LATENCY           (4)
+#define BL                      (16)
```

Comment:
We can just use BL_16 now since we updated the values.

Reply:
So I think at some point we will want to have a common location for this value, since we will only ever be using one BL value for the specified DRAM which is attached.  But if there was a common place where this was located, we can utilize that throughout the training process (and other calls) rather than hard coding it everywhere.  Not sure if this would be a configuration setting or if it is better to read it from the DRAM at boot to see what is connected.

Reply:
BL is not static. The BL can be set in MR1. I think for our case we will likely never use BL_32 for our training. I'm not sure it buys us anything, but regardless we have support for it. On this comment, I just wanted you to not have another BL MACRO that is the same as BL_16. This used to be necessary because BL_16 = 0. Now that BL_16 = 16. BL can be replaced by BL_16.

Reply:
Fixed.